### PR TITLE
Changed favicon

### PR DESCRIPTION
**Pull request by lord-cyclone100**
###### Before
Screenshot 2024-10-02 220928

###### After
Screenshot 2024-10-03 175833
Screenshot 2024-10-03 175854

## Description

changed favicon

## Limitations

none

## Checklist

- [+] my PR is focused and contains one wholistic change
- [+] I have added screenshots or screen recordings to show the changes

### CODE REVIEW

**Comment by lord-cyclone100** [comment]
@yurijmikhalevich 
This is in response to issue #44 and pr #50
I think now it is visible both n light and dark backgrounds.
Pls confirm. And if u merge pls add hacktoberfest-accepted label
also close my previous pr
Thanks!!

File: img/favicon.png
Comment (yurijmikhalevich) [inline img/favicon.png:1]: @lord-cyclone100, it is much better! Thank you 🔥 

I also want for @Misofser to comment when she has the time.

Comment (lord-cyclone100) [reply img/favicon.png:1]: ok @yurijmikhalevich 
@Misofser pls look into the matter when you have time

Comment (Misofser) [reply img/favicon.png:1]: @lord-cyclone100 Thank you for contributing. Great work! But have you thought about making the favicon more iconic? I think it's also not necessary to use letters. I recommend looking at the favicons of different websites you use. And look for some inspiration. Also, to make the favicon readable regardless of the theme, you could try using a background.

Comment (yurijmikhalevich) [reply img/favicon.png:1]: @Misofser, thank you for the review 😄 

@lord-cyclone100, if you can share a few options with us, we can get to the one we need quicker :) Trying something without the letters can be interesting.

Comment (lord-cyclone100) [reply img/favicon.png:1]: sure @yurijmikhalevich

Comment (Misofser) [reply img/favicon.png:1]: Sorry, it was typo. I wanted to use 'laconic' instead of 'iconic'. I meant that it can be more minimalistic 😄

**Pull request closed without merging**